### ISSUE TITLE
Using a non-bundled htslib version

### ISSUE THREAD

**Issue by samfux84** [arq5x/lumpy-sv#331]
Hi,

Is it possible to use a non-bundled htslib to build lumpy-sv 0.3.0 ?

Because using the bundled one requires to clone with "git clone --recursive", which makes the releases provided in https://github.com/arq5x/lumpy-sv/releases useless as they don't contain the content of the htslib directory.

We have several htslib versions installed on our cluster, and I prefer to build from a release tarball instead of using git clone --recursive.

Best regards

Sam

**Comment by samfux84** [comment]
Will removing

htslib:
        $(shell cd lib/htslib && autoreconf)
        cd lib/htslib && \
        ./configure --disable-bz2 --disable-lzma --enable-libcurl
        CFLAGS="$(CFLAGS) -DBGZF" $(MAKE) -C lib/htslib --no-print-directory CFLAGS="-DBGZF_MT"

from the Makefile and providing an external htslib break lumpy-sv ?

**Comment by ryanlayer** [comment]
You can also just remove the "htslib" from the "lumpy_filter: htslib" line

After that you need to edit lumpy-sv/src/filter/Makefile to point at the
htslib you want

**Comment by samfux84** [comment]
@ryanlayer: Thank you very much for your reply.

I tried building version 0.3.0 with the SPACK package manager that so far only supports version 0.2.13:

https://spack.readthedocs.io/en/latest/package_list.html#lumpy-sv

I could not manage to adapt the package such that it also builds version 0.3.0 fine, therefore I just installed it manually, following your documentation with git clone --recursive.

In any case thank you for being responsive and for providing help.

Best regards

Sam
